Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -18,22 +18,30 @@ def __init__(self, id, key, secret, url):
         self.id = id
```

Comment:
I've just attempted to deploy this manually into a new Azure Sentinel instance and I noticed that during the deployment the parameter names don't look great. I'm seeing things like

`Digitalshadows Account ID` rather than `Digital Shadows Account ID`

in `digitalshadowsARM.json` could we try changing these key names please?

```
        "digitalshadowsAccountID": {
            "type": "securestring",
            "defaultValue": ""
        },
        "digitalshadowsKey": {
            "type": "securestring",
            "defaultValue": ""
        },
        "digitalshadowsSecret": {
            "type": "securestring",
            "defaultValue": ""
        },
        "digitalshadowsUrl": {
            "type": "string",
            "defaultValue": ""
        },
        "historicaldays": {
```

And see if that fixes their naming when deploying? For instance, change: 

* `digitalshadowsAccountID` to `DigitalShadowsAccountID` 
* `digitalshadowsKey` to `DigitalShadowsKey`
* `digitalshadowsSecret` to `DigitalShadowsSecret`
* `digitalshadowsUrl` to `DigitalShadowsURL`
* `historicaldays` to `HistoricalDays`

this might necessitate changes to the data connector where it is importing those parameters from environment variables. I.e.

```
account_id = os.environ['digitalshadowsAccountID']
```